client/src/components/Carousel.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+// import MobileStepper from '@mui/material/MobileStepper';
+import Typography from '@mui/material/Typography';
+import ExpandCircleDownOutlinedIcon from '@mui/icons-material/ExpandCircleDownOutlined';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+// import KeyboardArrowLeft from '@mui/icons-material/KeyboardArrowLeft';
+// import KeyboardArrowRight from '@mui/icons-material/KeyboardArrowRight';
+import makeStyles from '@mui/styles/makeStyles';
+import type { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  topBanner: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '1rem 0 1rem 0',
+  },
+  bannerLeft: {
+    display: 'flex',
+  },
+  bannerRight: {},
+  carouselContent: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  carouselWrapper: {
+    margin: 'auto',
+    width: '80%',
+  },
+  questions: {
+    margin: '0.75rem 0 0 3rem',
+    color: 'gray',
+  },
+}));
+
+type CarouselProps = {
+  steps: { label: string; imgPath: string }[];
```

Comment:
nit picky, but can we make this look more like the data will be in the final form? aka...
```
// to be replaced by Asset
type Temp = {
  img: string,
  label: str, // alt
}

type Props = {
  cardGroups: Array<Temp[]>
}

// Home component data now looks like
const carouselSteps = [
  [
    {
      label: 'cat 1',
      imgPath: 'http://placekitten.com/200/200',
    },
    ...
```
so each child array here is the list of 3 cards/assets that we show. that you can map over in this component
lmk if that doesn't make sense